ocaml/testsuite/tests/translprim/comparison_table.heap.reference

```diff
@@ -96,161 +103,195 @@
        (function {nlocal = 0} x[nativeint] y[nativeint] : int
          (Nativeint.>= x y))
      eta_gen_cmp =
-       (function {nlocal = 0} prim prim stub (caml_compare prim prim))
+       (function {nlocal = 0} prim prim stub ignore assert all zero_alloc
+         : int (caml_compare prim prim))
      eta_int_cmp =
        (function {nlocal = 0} prim[int] prim[int] stub
-         (compare_ints prim prim))
+         ignore assert all zero_alloc : int (compare_ints prim prim))
```

Comment:
What is the source of this diff?

Reply:
It appears this was changed a while back: https://github.com/ocaml-flambda/flambda-backend/blame/main/ocaml/lambda/lambda.ml#L738

But somehow this test output was never regenerated?

Reply:
OK, I dug into this a little more.  The two files that changed are for the no-stack-allocation and no-flat-float-array configurations, which we don't actually build or test anymore with the full testsuite, but which can get run if you build an individual test.  Presumably you regenerated the output of these tests for other reasons, and that regenerated it for all configurations, even those we don't use.  Seems fine.